src/PockyBot.NET/Services/Pegs/PegCommentValidator.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Linq;
+
+namespace PockyBot.NET.Services.Pegs
+{
+    internal class PegCommentValidator : IPegCommentValidator
+    {
+        public bool IsPegValid(string comment, int? requireKeywords, string[] keywords, string[] penaltyKeywords)
+        {
+            if (requireKeywords == 1)
+            {
+                return keywords.Any(x => comment.IndexOf(x, StringComparison.InvariantCultureIgnoreCase) >= 0);
```

Comment:
Courtesy of Craig:

[Best practices for strings](https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings) say to use OrdinalIgnoreCase unless specifically needed otherwise.